app/models/planning_application.rb

```diff
@@ -78,6 +78,8 @@ class WithdrawOrCancelError < RuntimeError; end
     delegate :required?, to: :press_notice
     delegate :required?, to: :site_notice
   end
+  delegate :params_v1, to: :planx_planning_data, allow_nil: true
+  delegate :params_v2, to: :planx_planning_data, allow_nil: true
```

Comment:
as we don't have any `params_v2` related in this PR, I wouldn't add it. I would add it in the next PR which will use it